### DISCUSSION TITLE
Cor da capa do template

### DISCUSSION THREAD

**Discussion by Martafct** [joaomlourenco/novathesis#380] [Q&A]
Boa tarde! Espero que se encontre bem. 
Comecei agora a usar o template NOVAthesis e percebi que para o meu curso e Faculdade, (licenciatura em Química aplicada da FCT/NOVA), a capa tem de ser azul, em vez de verde. 

Procurei e não encontrei como fazer essa mudança. 
Para além disto, a capa aparece em Inglês e não é suposto no meu Projeto.

**Comment by joaomlourenco** [comment]
Please use english in this forum.
You have to switch the dissertation type from PhD to MSc.
Please take some time and:

- read the instructions provided in the template itself.
- take some time "studying" the relevant files and folders, e.g., did you look (I mean, really look) at the Config folder and *all the files* it contains?
